serving/docker/deepspeed.Dockerfile

```diff
@@ -65,6 +65,7 @@ ENV SAFETENSORS_FAST_GPU=1
 ENV NCCL_BLOCKING_WAIT=0
 ENV NCCL_ASYNC_ERROR_HANDLING=1
 ENV TORCH_NCCL_AVOID_RECORD_STREAMS=1
+ENV SERVING_FEATURES=vllm,lmi-dist
```

Comment:
last time when we introduce this feature, our bring your own `model.py` no longer works. Let's add a small bring your own model.py test and validate it

Reply:
I'll test with a custom model.py - was the issue that using a custom handler did not work at all due to this environment variable? or was there some other functionality that broke?

Reply:
I tested with a custom model.py (I copied the huggingface.py handler to a model.py) and it works.

But I also think that this logic here shouldn't be used if user provides a custom model.py. We don't inspect the custom model.py so it's hard to determine whether any of the configurations we set here actually make sense in that context. But since most people providing a custom model.py are doing so for pre/post processing logic (just extending our handler in a sense), I guess this is fine.

Reply:
Last time, it was a bug in detecting `rolling_batch`, it's not related to `model.py`, it's just a mismatch between `model.py` implementation vs detected rolling_batch mode

Reply:
I see. If the bug was in detecting rolling batch from the custom model.py, that seems like a model.py bug rather than a potential bug here